### DISCUSSION TITLE
Do I need to backup the name of my ord wallet?

### DISCUSSION THREAD

**Discussion by DanielHook15** [ordinals/ord#2016] [Q&A] [answered]
The standard name of an ord wallet is ord. However, I have selected a different name for my wallet when I created it using ord. 
Do I have to save the chosen wallet name?
Is it sufficient to have the 12-word mnemonic to be able to restore the wallet or is it necessary to remember also the name of the ord wallet?

**Comment by so7ow** [comment, marked as answer]
You don't need the name.
